src/compile_torture_tests.py

```diff
@@ -58,8 +58,6 @@ def run(c, cxx, testsuite, sysroot_dir, fails, out, config='wasm'):
                              'BINARYEN_METHOD="interpret-binary"'],
   }
 
-  assert os.path.isfile(c), 'Cannot find C compiler at %s' % c
```

Comment:
I'm assuming it's ok to not check here because it will fail just fine by itself in the call to compile?